one_fm/api/v1/authentication.py

```diff
@@ -351,3 +357,57 @@ def fetch_employee_checkin_list(from_date=None, to_date=None, limit=20, page_num
 		"number of checkin": len(check_list)
 	}
 	return response("Successfully Retrieved !", 200, data)
+
+@frappe.whitelist(allow_guest=True)
+def new_forgot_password(employee_id=None):
```

Comment:
What is the use of this method

Reply:
This checks if the employee exists, and returns the user_id, if it exists...The user_id is important as it is used to get the otp